server/src/main/java/com/flipkart/varadhi/web/handlers/TopicSchemaValidationHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.flipkart.varadhi.web.handlers;
+
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.validation.ValidationHandler;
+import io.vertx.json.schema.*;
+import lombok.extern.slf4j.Slf4j;
+
+import static io.vertx.json.schema.common.dsl.Keywords.maxLength;
+import static io.vertx.json.schema.common.dsl.Keywords.minLength;
+import static io.vertx.json.schema.common.dsl.Schemas.*;
+
+@Slf4j
+public class TopicSchemaValidationHandler implements ValidationHandler {
```

Comment:
As discussed, lets not add this as it will result in double deserialisation and validation logic being decoupled from the actual resource.